src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -207,10 +215,11 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
             OperatorSetParam memory operatorSetParam = _quorumOperatorSetParams[quorumNumber];
             {
                 uint32 numOperatorsForQuorum = indexRegistry.totalOperatorsForQuorum(quorumNumber);
-                require(
-                    numOperatorsForQuorum == operatorSetParam.maxOperatorCount + 1,
-                    "BLSIndexRegistryCoordinator.registerOperatorWithCoordinator: quorum has not reached max operator count"
-                );
+                // if the number of operators for the quorum is less than or equal to the max operator count, 
+                // then the quorum has not reached the max operator count
+                if(numOperatorsForQuorum <= operatorSetParam.maxOperatorCount) {
```

Comment:
we are enforcing this max operator count elsewhere, right?
writing this as a bit of a note -- am assuming we have a test for this, or else a plan to add one, at least.

Reply:
great catch, I don't think we are. I'll add logic for this